play-media-native/screens/AthletesListing.tsx

```diff
@@ -5,9 +5,13 @@ import { Athlete } from "../interfaces/athlete";
 import { getAllAthletes } from "../api/queries/getAthletes";
 import { useQuery } from "react-query";
 import { StatusBar } from "react-native";
+import { AnimatedFAB } from "react-native-paper";
+import { defaultStyle } from "../components/BottomFAB/BottomFAB";
```

Comment:
Maybe we need this style in a global styles folder, but for a later task probably.